app/controllers/project_users_controller.rb

```diff
@@ -28,4 +38,12 @@ def check_permission
     flash[:warning] = t ".not_permission"
     redirect_to project_path @project
   end
+
+  def destroy_project_user
+    if @project_user&.destroy
```

Comment:
trên method destroy check rồi
```suggestion
    if @project_user.destroy
```